src/elements/Step/StepContent.js

```diff
@@ -57,7 +57,11 @@ StepContent.propTypes = {
   /** Primary content of the StepTitle. Mutually exclusive with children. */
   title: customPropTypes.every([
     customPropTypes.disallow(['children']),
-    PropTypes.node,
+    PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.number,
+      PropTypes.object,
+    ])
```

Comment:
Will be there something like `customPropTypes.shorthand()`? It looks it might be

```jsx
PropTypes.oneOfType([
  PropTypes.element,
  PropTypes.number,
  PropTypes.object,
  PropTypes.string,
])
```